gen_config/CMakeLists.txt

```diff
@@ -42,10 +42,17 @@ add_definitions(
   -DTARGET_FRDM -DTARGET_Freescale -DTARGET_Freescale_EMAC -DTARGET_K64F -DTARGET_KPSDK_CODE
   -DTARGET_KPSDK_MCUS -DTARGET_KSDK2_MCUS -DTARGET_LIKE_CORTEX_M4 -DTARGET_LIKE_MBED -DTARGET_M4
   -DTARGET_MCUXpresso_MCUS -DTARGET_MCU_K64F -DTARGET_NAME=K64F -DTARGET_PSA -DTARGET_RELEASE
-  -DTARGET_RTOS_M4_M7 -DTOOLCHAIN_GCC -DTOOLCHAIN_GCC_ARM -DMBEDTLS_MD5_C -D__CMSIS_RTOS
+  -DTARGET_RTOS_M4_M7 -DTOOLCHAIN_ARM -DMBEDTLS_MD5_C -D__CMSIS_RTOS
```

Comment:
Should you not remove `-DTOOLCHAIN_ARM` ?

Reply:
fixed